### PR TITLE
[4.0] [UX] Sample Data success message

### PR DESCRIPTION
**Pull request by brianteeman**
Add an extra stage to the sampledata plugin just so that we display a message so the user knows the install of the sample data has finished

### Test instructions
Before this PR you get 3 success messages, 

After this PR you get a final success message as below

chrome_2018-06-27_18-51-58

### CODE REVIEW

File: plugins/sampledata/blog/blog.php [hunk 1]
```diff
@@ -952,6 +952,22 @@ public function onAjaxSampledataApplyStep3()
 		return $response;
 	}
 
+	/**
+	 * Final step to show completion of  sampledata.
```
Comment (Quy) [inline plugins/sampledata/blog/blog.php]: Remove extra space

File: plugins/sampledata/blog/blog.php [hunk 2]
```diff
@@ -952,6 +952,22 @@ public function onAjaxSampledataApplyStep3()
...
+	 *
+	 * @return  array or void  Will be converted into the JSON response to the module.
+	 *
+	 * @since __DEPLOY_VERSION__
```
Comment (Quy) [inline plugins/sampledata/blog/blog.php]: 2 spaces

File: administrator/language/en-GB/en-GB.plg_sampledata_blog.ini [hunk 3]
```diff
@@ -61,4 +61,5 @@ PLG_SAMPLEDATA_BLOG_STEP_SKIPPED="Step %1$u Skipped: '%2$s' is either not instal
 PLG_SAMPLEDATA_BLOG_STEP1_SUCCESS="Step 1: Articles done!"
 PLG_SAMPLEDATA_BLOG_STEP2_SUCCESS="Step 2: Menus done!"
 PLG_SAMPLEDATA_BLOG_STEP3_SUCCESS="Step 3: Modules done!"
+PLG_SAMPLEDATA_BLOG_STEP4_SUCCESS="Blog Sample Data has been successfully installed!"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_sampledata_blog.ini]: You previously did a PR to remove `successfully`.

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.plg_sampledata_blog.ini]: So I did :)

File: plugins/sampledata/blog/blog.php [hunk 4]
```diff
@@ -952,6 +952,22 @@ public function onAjaxSampledataApplyStep3()
...
 	}
 
+	/**
+	 * Final step to show completion of sampledata.
+	 *
+	 * @return  array or void  Will be converted into the JSON response to the module.
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 */
+
```
Comment (Quy) [inline plugins/sampledata/blog/blog.php]: Remove blank line

**Comment by Bakual** [comment]
I always assumed the progress bar being at 100% and the disappeared spinning Joomla logo gives enough clues that it finished. 😄 
But yeah, we can add that extra step to show a success message if that makes it more clear. Or we could adjust the success message from step 3.

**Comment by brianteeman** [comment]
@Bakual i think a separate message is best

**Review by Bakual — APPROVED** [review]
From a code review, this is fine.

**Comment by brianteeman** [comment]
@Bakual It came up while I was giving a demo. It is less obvious in this plugin that it is in the other one which has an extra message tagged onto the last step which says "please refresh the page"

**Comment by brianteeman** [comment]
Thanks

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**